src/prettytable/prettytable.py

```diff
@@ -511,6 +529,16 @@ def _validate_attributes(self, name, val):
     # ATTRIBUTE MANAGEMENT       #
     ##############################
 
+    @property
+    def columns_min_widths(self):
+        return self._columns_min_widths
+
+    @columns_min_widths.setter
+    def columns_min_widths(self, val):
+        val = [int(x) if x else 0 for x in val]
```

Comment:
And if we use `0` instead of `None`, we could do something like this and allow the normal `TypeError` for `int(None)`: 
```suggestion
        val = [int(x) for x in val]
```